documentation/asciidoc/computers/getting-started/setting-up.adoc

```diff
@@ -20,6 +20,15 @@ For the Raspberry Pi 4 Model B and Raspberry Pi 400, you should use the https://
 
 Finally, you'll need an xref:getting-started.adoc#sd-cards[SD card]; we recommend a minimum of 8GB micro SD card and use the https://www.raspberrypi.com/software/[Raspberry Pi Imager] to install an operating system onto it.
 
+[.whitepaper, title="Transitioning from Pi 3 to Pi 4", subtitle="", link=https://pip.raspberrypi.com/categories/685-whitepapers-app-notes/documents/RP-003475-WP/Transitioning-from-Pi-3-to-Pi-4.pdf]
```

Comment:
I don't think the "getting started" section is the right place for this. https://www.raspberrypi.com/documentation/computers/raspberry-pi.html is probably a better place, but I'm not sure _where_ on that page it should go.

Reply:
Agreed. Just because a white paper exists doesn't mean we have to find a place to reference it. The point of them is that they're supposed to be on topics outside the scope of the documentation. So I'd be surprised if we can reference them all. This one for instance probably isn't needed.

Reply:
Yeah, cut this one.